Triphouse-project/index.html

```diff
@@ -8,12 +8,72 @@
     <link rel="preconnect" href="https://fonts.gstatic.com" />
     <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet" />
     <link rel="stylesheet" href="./style.css" />
-    <title>Document</title>
+    <title>Triphouse</title>
 </head>
-
 <body>
     <header>
```

Comment:
![2021-04-19_22h22_52](https://user-images.githubusercontent.com/19935417/115291802-d905d280-a15d-11eb-8bb0-f34359c11019.png)
The background image should cover the whole page width.

Reply:
Fixed background image